docs/sources/github/github.yaml

```diff
@@ -78,6 +116,46 @@ endpoints:
       username:
         type: "string"
         format: "reversible-pseudonym"
+  - pathTemplate: "/api/{enterpriseServerVersion}/users/{username}"
```

Comment:
why cloning with diff template? Just outlining options, not sure what's best:
- different file for enterprise cloud and server
- use regex "(/api/{enterpriseServerVersion)?/users/{username}"

Reply:
Because I had in the task that sourceKind should be the same, so only way to add more rules was to include them in the same file. Tried with regex but that would complicate a bit the rules, so just split in two files -with different sourceKindId but same scopeId